src/lib/router.js

```diff
@@ -105,8 +107,8 @@ const renderRoute = (props, route) => {
         Nothing: () => 'Wallet',
         Just: (wal) =>
             wal.value instanceof EthereumWallet
-          ? wal.value.address
-          : addressFromSecp256k1Public(wal.value.publicKey)
+          ? (<span className="text-mono">{wal.value.address}</span>)
+          : (<span className="text-mono">{addressFromSecp256k1Public(wal.value.publicKey)}</span>)
```

Comment:
Can this not be wrapped outside of the `? :`? idk how jsx works.

Reply:
It's a bit uglier that way IMO; I try to keep the amount of logic inside JSX brackets minimal.